<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SleepTimer</class>
 <widget class="QWidget" name="SleepTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>262</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Academy Engraved LET</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SleepTimer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(28, 88, 162);</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>How long before your computer shuts down ?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 1px solid #FFFFFF;
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #E9BB5F;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>67</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #E9BB5F;</string>
   </property>
   <property name="text">
    <string>SleepTimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="byMe">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
    </font>
   </property>
   <property name="text">
    <string>by Bastien Schwitz</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
